sequence.go

```diff
@@ -654,28 +659,36 @@ func findSequencesOnDisk(path string, opts ...FileOption) (FileSequences, error)
 
 		frame, _ := strconv.Atoi(match[2])
 		key := [2]string{match[1], match[3]}
-		frames, ok := seqs[key]
+		seq, ok := seqs[key]
+		frameWidth := len(match[2])
 		if !ok {
-			frames = []int{frame}
-			padMap[key] = padder.PaddingChars(len(match[2]))
+			seq = seqInfo{
+				Frames:   []int{frame},
+				Padding:  padder.PaddingChars(frameWidth),
+				MinWidth: frameWidth,
+			}
 		} else {
-			frames = append(frames, frame)
+			seq.Frames = append(seq.Frames, frame)
+			if seq.MinWidth > frameWidth {
+				seq.MinWidth = frameWidth
+				seq.Padding = padder.PaddingChars(frameWidth)
+			}
 		}
-		seqs[key] = frames
+		seqs[key] = seq
```

Comment:
Thanks for these fixes! 

Do you think this code could now be simplified to something like this?

```go
		seq.Frames = append(seq.Frames, frame)
		seq.Padding = padder.PaddingChars(frameWidth)

		if seq.MinWidth > frameWidth {
			seq.MinWidth = frameWidth
		}

		seqs[key] = seq
```

You basically do all the same stuff regardless of whether the struct was found in the map since its always a copy and ready to be used. If you switched it to storing a pointer in the slice then you would need that `ok` check to allocate a new one and add it to the map.